development/circular-deps.ts

```diff
@@ -94,6 +94,19 @@ const MADGE_CONFIG = {
   excludeRegExp: IGNORE_PATTERNS.map(globToRegExp),
   tsConfig: 'tsconfig.json',
   webpackConfig: 'webpack.config.js',
+  detectiveOptions: {
```

Comment:
Please remove the .madgerc file as it is no longer used!

Reply:
I went the other way. I moved this config to a madgerc file so we can still use the CLI to do things